README.md

```diff
@@ -1,25 +1,25 @@
 
 [The Intergovernmental Panel on Climate Change](https://report.ipcc.ch/sr15/pdf/sr15_spm_final.pdf) (IPCC) declares 2030 as the tipping point for irreversible damage to our planet. It is imperative for industry leaders such as Microsoft to respond to this emergency as soon as possible and set an example for other tech companies. This is humanity's biggest challenge, requiring the contributions from corporations, organizations, and government to enact systemic change.
 
 The youth have realized that climate change is a threat insurmountable by individual action and have [called](https://www.theguardian.com/commentisfree/2019/may/23/greta-thunberg-young-people-climate-strikes-20-september) upon the rest of the world to support them. We must use the Global Climate Strike, inspired by 16-year-old Swedish climate activist [Greta Thunberg's school strikes](https://www.fridaysforfuture.org/), as an opportunity to stand together and demand the transformation that our world needs. Doing so is not only taking responsibility for the mistakes that future generations will have to pay for but also the only viable way forward.
 
-But first, we have to acknowledge our company's role in our warming planet. 
+Our company should reconsider its technology support of oil and gas exploitation, and should have a deliberate plan to withdraw our innovation from that sector, as a matter of principal.
 
-Microsoft makes millions of dollars in profits by helping fossil fuel companies extract more oil. In this week's company-wide all-hands, our CEO, Satya Nadella, defended our contracts with Big Oil by suggesting that our technology helps them in their R&D efforts to develop sustainable energy. Yet, this is in clear contradiction to the ambitions of fossil fuel-CEOs and even other Microsoft leaders: Azure Executive Tom Keane, in discussing [Microsoft's multimillion dollar contract with Chevron](https://news.microsoft.com/transform/chevron-fuels-digital-transformation-with-new-microsoft-partnership/), asked "How can we [..] more efficiently do oil exploration? Or more efficiently do sensor management from an offshore oil rig?" 
+One could defend our contracts with oil and gas by suggesting that our technology helps in their R&D efforts to develop sustainable energy. Yet, Azure Executive Tom Keane, in discussing [Microsoft's multimillion dollar contract with Chevron](https://news.microsoft.com/transform/chevron-fuels-digital-transformation-with-new-microsoft-partnership/), asked "How can we [..] more efficiently do oil exploration? Or more efficiently do sensor management from an offshore oil rig?" 
```

Comment:
"One could defend" lends legitimacy to that defense. This phrasing works against the purpose of this document. I think the original wording should be left in place.

Reply:
I was also trying to remove an inferred quote from the CEO which was given to an _internal_ audience. Our Yammer community says its content is not for sharing.

Reply:
I think the comment is a really crucial part of this conversation, regardless of what the Yammer guidelines state. It should stay.